app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -31,6 +35,12 @@
 import com.google.android.exoplayer2.ui.SimpleExoPlayerView;
 
 import java.util.Calendar;
+import java.util.Collection;
```

Comment:
ERROR: Unused import: java.util.Collection. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports